nalgebra-lapack/src/qz.rs

```diff
@@ -0,0 +1,325 @@
+#[cfg(feature = "serde-serialize")]
+use serde::{Deserialize, Serialize};
+
+use num::Zero;
+use num_complex::Complex;
+
+use simba::scalar::RealField;
+
+use crate::ComplexHelper;
+use na::allocator::Allocator;
+use na::dimension::{Const, Dim};
+use na::{DefaultAllocator, Matrix, OMatrix, OVector, Scalar};
+
+use lapack;
+
+/// Generalized eigendecomposition of a pair of N*N square matrices.
```

Comment:
Would be good if docs were more specific. How precisely is the QZ decomposition defined?